package.json

```diff
@@ -35,7 +35,6 @@
   "dependencies": {
     "@sindresorhus/slugify": "^0.6.0",
     "detect-port": "^1.2.3",
-    "docz-core": "^0.12.5",
```

Comment:
Would it be better to make this a peer dependency instead?  Although if we accept a function then we don't really have a dependency as long as the function signature hasn't changed in a breaking manner.

Reply:
+1 for peer-dependency